pants.travis-ci.toml

```diff
@@ -36,6 +36,9 @@ timeout_maximum = 590
 # NB: See `test.pytest`.
 timeout_maximum = 590
 
+[pytest]
+# NB: We don't set a max test timeout for v2, because tests run concurrently.
```

Comment:
Why have this at all?

Reply:
To explain the lack of setting it relative to v1, which we do set it for.

Reply:
Sure, but v1 is going away ;)